web/src/components/ToggleButton.tsx

```diff
@@ -57,7 +62,7 @@ export default function ToggleButton({
             key={`group-item-${key}`}
             value={value}
             onClick={() => onToggle(value)}
-            data-test-id={`toggle-button-${dataTestId}`}
+            data-test-id={`toggle-button-${dataTestId ?? value}`}
```

Comment:
I think we'll have to remember to use quotes when referencing the id if it defaults to value and value has spaces etc.

Reply:
Hmm good point, should we remove the fallback and just declare the IDs directly even if they are the same value?